File: JavaScript/Json.js

// JSONオブジェクト
var json = [
    {
        "name": "Honda",
        "age": "21",
        "address": "Tokyo"
    },
    {
        "name": "Yamaha",
        "age": "35",
        "address": "Osaka"
    },
]

console.log(json[0].name)
console.log(json[1].age)

// 要素の追加
json.push({
    "name": "Kawasaki",
    "age": "42",
    "address": "Hakana"
})
console.log(json[2].address)

// 要素の削除
delete json[2].address
console.log(json[2].address)
console.log(json[2].name)
delete json[2]
console.log(json[2])

// 文字列からJSONへ
var str = JSON.stringify({ x: 1, y: 100 })
console.log(str)

// 文字列からJSONへ
json = JSON.parse(str)
console.log(json)

File: Python/Json.py
# coding: utf-8

import json

# 文字列からJSONへ
str1 =  "[{\"name\": \"Honda\", \"age\": 21, \"address\": \"Tokyo\"}, {\"name\": \"Yamaha\", \"age\": 35, \"address\": \"Osaka\"}]"
json1 = json.loads(str1)
print(json1)

# アクセス
print(json1[0]["name"])
print(json1[1]["age"])
for j in json1:
    print(j)
    print(j["name"])

# 追加と削除
json1.append({"name": "Kawasaki", "age": 99, "address": "Nagoya"})
print(json1)
del json1[2]
print(json1)

# JSONから文字列へ
str1 = json.dumps(json1)
print(str1)
